packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/CreateFolderControls.tsx

```diff
@@ -52,35 +79,30 @@ export const CreateFolderControls = (): React.JSX.Element => {
     <>
       <Title />
       <Exit onClick={() => handleUpdateState({ type: 'EXIT' })} />
-      <Primary {...primaryProps} />
-
+      {result?.status === 'SUCCESS' ? (
+        <Message variant="success">Folder created.</Message>
+      ) : null}
       <Target.Field.Container>
         <Target.Field.Label htmlFor="folder-name-input">
           Enter folder name:
         </Target.Field.Label>
         <Target.Field.Input
           disabled={isLoading}
+          aria-invalid={fieldValidationError ? 'true' : undefined}
+          aria-describedby="fieldError"
           type="text"
           id="folder-name-input"
+          onBlur={handleFieldValidation}
+          onFocus={() => setFieldValidationError(undefined)}
           ref={inputRef}
         />
-        {fieldValidationError ? <span>{fieldValidationError}</span> : null}
-        <Target.Field.Button
-          onClick={() => {
-            if (!inputRef.current?.value?.endsWith('/')) {
-              setFieldValidationError(
-                'Folder name must end with a "/" character'
-              );
-              return;
-            }
-            // clear error
-            setFieldValidationError(undefined);
-            setData(() => inputRef.current?.value ?? '');
-          }}
-        >
-          Set Folder Name
-        </Target.Field.Button>
+        {fieldValidationError ? (
+          <Target.Field.Error id="fieldError">
```

Comment:
Is camelCase standard for `id` values?

Reply:
I think our usage has been kind of all over the place. I'm not sure there is a standard given it's not needed to be human readable. 

In any case, it should proooooobably use useStableId utility (or just React.useId()) when we're passing id's meant for accessibility.

Reply:
Yeah that makes sense, can we please add a backlog item for auditing/moving to stable/unique ids?

Reply:
Added to our backlog